aries_cloudagent/wallet/routes.py

```diff
@@ -757,6 +786,90 @@ async def wallet_set_did_endpoint(request: web.BaseRequest):
         return web.json_response({"txn": transaction.serialize()})
 
 
+@docs(tags=["wallet"], summary="Create a EdDSA jws using did keys with a given payload")
+@request_schema(JWSCreateSchema)
+@response_schema(WalletModuleResponseSchema(), description="")
+async def wallet_jwt_sign(request: web.BaseRequest):
+    """
+        Request handler for jws creation using did.
+
+    Args:
+        "headers": { ... },
+        "payload": { ... },
+        "did": "did:example:123",
+        "verificationMethod": "did:example:123#keys-1"
+        with did and verification being mutually exclusive.
+    """
+    context: AdminRequestContext = request["context"]
+    body = await request.json()
+    did = body.get("did")
+    if did is None:
+        did = body.get("verificationMethod").split("#", 1)[0]
```

Comment:
`body.get()` will return a `None` if `verificationMethod` is absent, which will raise an exception on `.split`.

Reply:
I see, I will fix the unsafe accesses.